src/packages/components/HomePage/CreateElection.tsx

```diff
@@ -0,0 +1,72 @@
+import Button from "@components/Button";
+import { addElection } from "@packages/DAO/elections.dao";
+import { ElectionStatus } from "@prisma/client";
+
+export default async function CreateElection() {
+  const electionsState = () => {
+    const elections = [...Object.values(ElectionStatus)];
+    return elections;
+  };
+  return (
+    <div className="flex pr-10 pl-10 pt-10 pb-10 flex-col gap-14 justify-center ring-1 ring-inset ring-gray-300 rounded-md border-0 py-1.5 pl-7 pr-20">
+      <h1 className="font-bold text-sl w-full align">Crie uma eleição</h1>
+      <form action={addElection}>
+        <div className="grid grid-cols-1 gap-x-6 gap-y-8 sm:grid-cols-6 ">
+          <div className="sm:col-span-10">
+            <label
+              htmlFor="username"
+              className="block text-sm font-medium leading-6 text-gray-900"
+            >
+              Nome
+            </label>
+            <div className="mt-2">
+              <div className="flex rounded-md shadow-sm ring-1 ring-inset ring-gray-300 focus-within:ring-2 focus-within:ring-inset focus-within:ring-indigo-600 sm:max-w-md">
+                <input
+                  type="text"
+                  name="electionName"
+                  id="username"
+                  className="block w-full rounded-md border-0 py-1.5 pl-7 pr-20 text-gray-900 ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"
```

Comment:
This padding inside the input seems weird 👀 

![Captura de Tela 2024-03-20 às 19 45 34](https://github.com/podcodar/voting-system/assets/14007745/9b07da2d-a926-46f7-b0b8-e89ce5dc40dc)